src/layout/constraint.rs

```diff
@@ -15,8 +15,10 @@ use itertools::Itertools;
 ///
 /// 1. [`Constraint::Fixed`]
 /// 2. [`Constraint::Min`] / [`Constraint::Max`]
```

Comment:
Is it worth splitting these too?

Reply:
`Min` and `Max` are equal priority because I don't expect users to do something like `[Min(100), Max(0)]` and expect a stable solution. But because I chose `Min`'s rank as higher than `Max`, I've updated the docstring to indicate that `Min` has a higher priority.